.github/workflows/test.yml

```diff
@@ -157,7 +157,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup-project
       - uses: mansagroup/nrwl-nx-action@v3
-        if: ${{matrix.projectName == '@novu/application-generic' }}
+        if: ${{matrix.projectName == '@novu/application-generic' || matrix.projectName == '@novu/notification-center'}}
```

Comment:
when there are the changes in the `notification-center` package then run the tests pipeline: lint, build, test